flink-cdc-runtime/src/test/java/org/apache/flink/cdc/runtime/operators/transform/UnifiedTransformOperatorTest.java

```diff
@@ -917,6 +917,52 @@ public void testMetadataAndCalculatedTransform() throws Exception {
                 .destroyHarness();
     }
 
+    @Test
+    public void testMetadataTransformIncludeMetaColumnString() throws Exception {
+        TableId tableId = TableId.tableId("my_company", "my_branch", "schema_nullability");
+        UnifiedTransformTestCase.of(
+                        tableId,
+                        "id, name, age, id + age as computed, __table_name__ as metaCol1, '__table_name__' as metaCol2, '__namespace__name__schema__name__table__name__' as metaCol3, UPPER(__schema_name__) as metaCol4",
```

Comment:
could you also add `__namespace_name__ as metaColNameSpaceName`, `__schema_name__ as metaColSchemaName` in this test?

Reply:
done